src/viam/sdk/module/handler_map.hpp

```diff
@@ -11,10 +11,12 @@ class HandlerMap_ {
    public:
     HandlerMap_();
     void add_model(Model model, RPCSubtype subtype);
-    std::unordered_map<RPCSubtype, std::vector<Model>> handles;
 
-    viam::module::v1::HandlerMap to_proto();
+    const viam::module::v1::HandlerMap to_proto() const;
```

Comment:
good catch, will update

Reply:
FYI returning a `const x` from a function doesn't constrain the caller in any way since they implicitly get a copy. There are clang-tidy rules to warn about this. I recommend dropping the first `const` here and in any similar decls.